spec/view_component/storybook/stories_spec.rb

```diff
@@ -16,6 +16,30 @@
       )
     end
 
+    it "converts kwargs" do
+      expect(KwargsComponentStories.to_csf_params).to eq(
+        title: "Kwargs Component",
+        stories: [
+          {
+            name: :default,
+            parameters: {
+              server: { id: "kwargs_component/default" }
+            },
+            args: {
+              message: "Hello World!",
+              param: 1,
+              other_param: true,
+            },
+            argTypes: {
+              message: { control: { type: :text }, name: "Message" },
+              param: { control: { type: :number }, name: "Param" },
+              other_param: { control: { type: :boolean }, name: "Other Param" },
+            }
+          }
+        ]
+      )
+    end
```

Comment:
Is there anything specific about `to_csf_params` and kwargs that we're worried about testing here?

Reply:
I wanted to showcase that storybook would create the controls even if they aren't used on the component since it accepts kwargs